src/cunumeric/cudalibs.cu

```diff
@@ -194,10 +194,10 @@ cublasHandle_t CUDALibraries::get_cublas()
 {
   if (nullptr == cublas_) {
     CHECK_CUBLAS(cublasCreate(&cublas_));
-    const char* disable_tensor_cores = getenv("CUNUMERIC_DISABLE_TENSOR_CORES");
-    if (nullptr == disable_tensor_cores) {
-      // No request to disable tensor cores so turn them on
-      cublasStatus_t status = cublasSetMathMode(cublas_, CUBLAS_TENSOR_OP_MATH);
+    const char* fast_math = getenv("CUNUMERIC_FAST_MATH");
```

Comment:
I chose this against `CUNUMERIC_DISABLE_TENSOR_CORES`, because if we don't set anything then cuBLAS will stil use tensor cores, but only if that wouldn't result in precision loss.